### ISSUE TITLE
Operand overload

### ISSUE THREAD

**Issue by ni-maxkoehler** [Remillard/VHDL-Mode#128]
Syntax highlighting for operand overload is not working.
![grafik](https://user-images.githubusercontent.com/27268149/56642456-c1cba180-6677-11e9-9956-07ba34cedb09.png)

**Comment by Remillard** [comment]
Thanks for letting me know!  I'll look into it.

**Comment by Remillard** [comment]
Believe I've got a fixed staged for this now.  Should be released fairly soon as soon as I test some other things in the release.

**Comment by ni-maxkoehler** [comment]
Thanks for taking care about this.
I had a quick look into the change. Does the regular expression captures function names like "and" as well?

**Comment by Remillard** [comment]
Mmm, probably not if overloading the logical operators requires the use of double quotes.  I looked through the abbreviated LRM in the Ashenden book, and then glanced at the section for operator overloading.  It was somewhat obscured if overloading those operators required quotes and I figured if it doesn't then the usual valid identifier match would work.  Easy enough to add though so I can do that.  I'm off work for retrieving kid from college today so don't know if I'll get to it today, maybe over the weekend.
